src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -1398,6 +1398,23 @@ public List<String> getHvdcLinesIds(UUID networkUuid, String variantId, List<Str
         }
     }
 
+    public Object getBranchOrThreeWindingsTransformer(UUID networkUuid, String variantId, String equipmentId) {
+        Network network = getNetwork(networkUuid, PreloadingStrategy.NONE, variantId);
+        Line line = network.getLine(equipmentId);
+        if (line != null) {
+            return toMapData(line, true);
```

Comment:
Are you sure we want busOrBusbarSections ?

Reply:
I don't know I kept the previous behaviour, I remove it ?

Reply:
I removed it from this endpoint